Client/Frontend/Brave Notifications/BraveNotification.swift

```diff
@@ -0,0 +1,37 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+
+public struct BraveNotificationPriority: Comparable {
```

Comment:
Same comment about `public`

Reply:
I have double checked `AdsNotificationHandler` and `BraveNotificationPresenter` do not need to be public. Hence, all the unnecessary public classes are now removed.